packages/oceanfront/src/scss/_badge.scss

```diff
@@ -1,37 +1,48 @@
 .of-badge {
+  --badge-v-pad: 0.5em;
+  --badge-h-pad: 0.5em;
   background: $bg-badge;
-  display: inline-block; 
+  display: inline-block;
   border-width: $border-width-badge;
   border-style: solid;
-  border-color: var(--border-color-badge); 
+  border-color: var(--border-color-badge);
   color: var(--color-badge);
   border-radius: $border-radius-badge;
-  margin: 0 .25em .5em 0.25em;
+  margin: 0 0.25em 0.5em 0.25em;
   font-weight: $font-weight-badge;
   font-size: $font-size-badge;
   min-width: 2ex;
   min-height: 2ex;
   line-height: $line-height-badge;
-  padding: $pad-v-badge $pad-h-badge;
+  padding: var(--badge-v-pad) var(--badge-h-pad) !important;
```

Comment:
Why's the `!important` in here? I'd really rather avoid that.

Reply:
This is because if the badge is inside the of-field, then used padding of this field 
I'll find another way

Reply:
@andrewwhitehead Done